Pages/Index.cshtml

```diff
@@ -7,23 +7,26 @@
 <div class="text-center">
     <h1 class="display-4">Welcome</h1>
     <table class="table">
-        <tr>
-            <th>Zip Code</th>
-            <th>Latitude</th>
-            <th>Longitude</th>
-            <th>Power Usage</th>
-            <th>Vehicle Count</th>
-        </tr>
-
-        @foreach (NeighbourhoodRank.Affluence affluence in affluences)
-        {
+        <thead class="thead-dark">
             <tr>
-                <td>@affluence.Zip</td>
-                <td>@affluence.Latitude</td>
-                <td>@affluence.Longitude</td>
-                <td>@affluence.PowerUsage</td>
-                <td>@affluence.VehicleCount</td>
+                <th>Zip Code</th>
```

Comment:
This makes sense... the titles will be static, and th is the correct HTML tag for table heading.